pkg/helm/helm.go

```diff
@@ -95,6 +97,9 @@ func (h *Helm) getReleasesVersionTwo() error {
 		if release.Version != deployed.Version {
 			continue
 		}
+		if h.Namespace != "" && release.Namespace != h.Namespace {
```

Comment:
I think this would give a larger performance benefit if it happened before line 93 when it's hitting the kube api again.

Reply:
good call. Thanks!